src/localstack/mod.rs

```diff
@@ -0,0 +1,109 @@
+use testcontainers::{core::WaitFor, Image};
+use std::collections::HashMap;
+
+const NAME: &str = "localstack/localstack";
+const TAG: &str = "3.0";
+const DEFAULT_WAIT: u64 = 3000;
+
+/// This module provides [LocalStack](https://www.localstack.cloud/) (Community Edition).
+/// 
+/// Currently pinned to [version `3.0`](https://hub.docker.com/layers/localstack/localstack/3.0/images/sha256-73698e485240939490134aadd7e429ac87ff068cd5ad09f5de8ccb76727c13e1?context=explore)
+/// 
+/// # Environment variables
+/// 
+/// For configuration, LocalStack uses environment variables. You can go [here](https://docs.localstack.cloud/references/configuration/)
+/// for the full list.
+/// 
+/// You can set a variable using the `with_environment_variable()` method.
+/// 
+/// ```
+/// # use testcontiners_modules::localstack::LocalStack;
+/// let image = LocalStack::default().with_environment_variable("DEBUG", "1");
+/// ```
+/// 
+/// No environment variables are required.
+#[derive(Default, Debug)]
+pub struct LocalStack {
+    env_vars: HashMap<String, String>
+}
+
+impl LocalStack {
+    /// Set an environment variable on the LocalStack image.
+    pub fn with_environment_variable(
+        self,
+        var_name: impl Into<String>,
+        value: impl Into<String>
+    ) -> Self {
+        let mut env_vars = self.env_vars;
+        env_vars.insert(var_name.into(), value.into());
+        Self { env_vars }
+    }
+}
```

Comment:
Usually, we don't enrich the images with so common methods, only some specific to image to make it easier to startup.

To add environment argument it's usually recommended to use `RunnableImage::with_env_var`: https://docs.rs/testcontainers/0.15.0/testcontainers/core/struct.RunnableImage.html (it has number of useful common methods)

Thus, code looks like:
```rs
let image = LocalStack::default().into().with_env_var("..", "..");

```

```suggestion
```

Reply:
Ahh. Didn't realize this was built-in. Thanks!

Reply:
Method removed and I have updated the documentation / examples.